docs/CONTRIBUTING.md

```diff
@@ -24,6 +24,7 @@ It is important to note that there are **two types of PRs** for this repository:
 - Consider using
   [conventionalcommits.org](https://www.conventionalcommits.org/en/v1.0.0/)'s
   rules for creating explicit and meaningful commit messages.
+- Update the version in the `==UserStyle==` header of the css-file. This is to enable automatic updates for users.
```

Comment:
`This is to enable automatic updates for users.` This isn't exactly true, it would be better if it was rephrased to some form of version control.

Reply:
According to https://github.com/openstyles/stylus/issues/39#issuecomment-345953794 while not automatic it makes the style able to be updated via the update url. Guess the phrasing automatic is a bit off, but it still enables updating without reinstall of the style.

Reply:
I know of it allowing updates it was more about the phrasing of `automatic` rather then the updates.

Reply:
Rephrased to: "This is to enable version control of the style."